SmartManager.cpp

```diff
@@ -0,0 +1,39 @@
+#include "SmartManager.h"
+
+void SmartManager::loopall() {
+    for (auto it = loopables.begin(); it != loopables.end(); ++it) {
+        if ((*it) != NULL) {
+            (*it)->loop();
+        }
+    }
+}
+
+LinkedList<Loopable*> SmartManager::loopables = LinkedList<Loopable*>([](const Loopable* ptr) {
```

Comment:
make this as a field instance(so as devices and sensors) of SmartManager instead of global var